BUILD.md

```diff
@@ -19,11 +19,11 @@ These additional packages are needed for building the components in this repo.
 
 Ubuntu 16.04 LTS and 18.04 have been tested with this repo.
 
-Continuous integration tools use [CMake 3.12.4](https://github.com/Kitware/CMake/releases/tag/v3.12.4) for Linux
+[CMake 3.10.2](https://cmake.org/files/v3.10/cmake-3.10.2-Linux-x86_64.tar.gz) is recommended.
 
 ```
 # Dependencies from included submodule components
-sudo apt-get install git cmake build-essential bison libx11-xcb-dev libxkbcommon-dev libwayland-dev libxrandr-dev libxcb-randr0-dev
+sudo apt-get install git build-essential bison libx11-xcb-dev libxkbcommon-dev libwayland-dev libxrandr-dev libxcb-randr0-dev
```

Comment:
It's convenient to have one or two apt-get commands that I can copy and past to install everything I need to build the repo. It's too bad we can't include cmake in such a command line.